File: src/Feedback.Analyzer.Backend/Feedback.Analyzer.Application/Organizations/Commands/UpdateOrganizationCommand.cs
using Feedback.Analyzer.Application.Organizations.Models;
using Feedback.Analyzer.Domain.Common.Commands;

namespace Feedback.Analyzer.Application.Organizations.Commands;

/// <summary>
/// Represents a command to modify an existing organization's information.
/// </summary>
public class UpdateOrganizationCommand : ICommand<OrganizationDto>
{
    /// <summary>
    /// Contains the updated organization data. 
    /// </summary>
    public OrganizationDto Organization { get; init; } = default!; 
}

File: src/Insight.Box.FrontEnd/src/modules/organizations/models/UpdateOrganizationCommand.ts
import {Command} from "@/infrastructure/models/command/Command";
import type {Organization} from "@/modules/organizations/models/Organization";

export class  UpdateOrganizationCommand extends Command{

    public organization:Organization;

    public constructor(organization: Organization) {
        super();
        this.organization = organization;
    }

}